<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Order Items</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>631</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QPushButton" name="addCartButton">
     <property name="geometry">
      <rect>
       <x>522</x>
       <y>210</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Add To Cart</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="menuTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>609</width>
       <height>181</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <column>
      <property name="text">
       <string>Item Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Category</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>People per serving</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Price</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>631</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Filter Items</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>611</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Item Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="itemNameLine"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Category</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="categoryBox"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>OR</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="searchButton">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>170</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>631</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Cart</string>
    </property>
    <widget class="QTableWidget" name="cartTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>611</width>
       <height>171</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>150</number>
     </attribute>
     <column>
      <property name="text">
       <string>Item Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Category</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Quantity</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Unit Price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Total Price</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QPushButton" name="checkOutButton">
    <property name="geometry">
     <rect>
      <x>512</x>
      <y>710</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Go To Checkout</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>710</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>652</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
